### ISSUE TITLE
Wierd error on windows 10

### ISSUE THREAD

**Issue by StuntsPT** [stuckyb/seqtrace#12]
Hi, I'm getting a strange error when running SeqTrace on windows 10. After extracting the `.zip` file and running the `.exe` I get a popup window with the following message:
"Failed to execute script run_seqtrace"
(Screenshot attached)

After a web search, I **suspect** this is related to pyinstaller not bundling all the assets correctly.
I cannot reproduce the issue on a VM with Windows 7. However, I should note that the VM has among other things, python installed, whereas the Windows 10 machine does not.

Do you have any other hints on how to get it to work from the binary? I will also try running it from source and report back, but I thought you might be interested in this.

![Screenshot](https://user-images.githubusercontent.com/1201571/64686824-3bc2c280-d481-11e9-8308-40ab60cb025e.PNG)

**Comment by stuckyb** [comment]
Thanks for the report!  I have definitely tested the binary on Windows 10, so I am not sure what might be causing the error you are seeing.  You are probably correct that there is something missing from the binary.  I have been working on a new release of SeqTrace that is almost ready to go, and I plan to redo the binary build process for that release, too, so I hope that will fix this problem.  I will try to get that all ready ASAP.

**Comment by StuntsPT** [comment]
Thank you for the quick reply!
I read you are dropping GTK. Just out of curiosity, which toolkit are you moving `seqtrace` into?

**Comment by stuckyb** [comment]
I'm not dropping GTK, exactly -- I am dropping the PyGTK library and GTK 2 to move to PyGObject and GTK 3.  Even though it is still GTK, it is a non-trivial change.

**Comment by StuntsPT** [comment]
Wow, that is quite an endavour. I have dabbled in it some year ago for a project, but ended up using Qt instead. I truly wish you the best of luck!
Also, marginally related, I suppose that move to PyGObject is also a required step in migrating to python3, right?
If/when you decide to move to python3, feel free to ping me for assistance. It sounds like a fun project I can help tackle during holiday season.

**Comment by stuckyb** [comment]
Correct, the move to PyGObject is a prerequisite for moving to Python 3 (and probably the biggest hurdle to clear).  If I started the project from scratch today, I'd probably use PyQt for the graphical toolkit.  But, GTK is still a very decent toolkit, and although moving to PyGObject/GTK 3 is definitely a pain, migrating to PyQt would be even worse.\
Thanks for the offer to help with an eventual Python 3 migration!  When I finally get around to that, I'll keep you mind!

**Comment by StuntsPT** [comment]
Yes, better to move from PyGTK to PyGObject than a full rewrite in PyQt.
I'll be watching the repository, and waiting for when you are ready to start the port. =-)

**Comment by StuntsPT** [comment]
I have figured out the original issue.
It is not due to missing dependencies from `pyinstaller`.
It's a simple issue with non-ascii characters. In the Windows 10 machine, the localization is set to "Portuguese". This means that the directory where I was trying `seqtrace` is not called "Downloads", bur rather "Transferências" (note the `ê`). Whenever there is a non ASCII character in the path `seqtrace` will fail to run.
Furthermore, I cannot import sequence trace files that exist in paths containing non-ascii characters.

I suppose this problem will solve itself when porting to python3, which is way better than python2 when handling non-ascii encodings.

So unless you are aware of a quick solution to solve the non-ascii issue, I suggest just writing a warning the the docs, while the port is "baking". And maybe leave the issue open until then.

**Comment by stuckyb** [comment]
Thanks so much for tracking that down!  That makes a lot of sense.  I suspect you are correct that porting to Python 3 will solve this problem because of Python 3's vastly superior unicode support.  I might still take a look at this and see if about a fix prior to the move to Python 3, because it seems like this could be a common problem.

**Comment by StuntsPT** [comment]
You're welcome. This probably affects every installation on non-english Windows systems.
I have just tested on ArchLinux and I can't reproduce the problem, so I suppose this is Windows specific (I don't have a mac to try it there).
